### ISSUE TITLE
Unable to find your account on Zoom. 

### ISSUE THREAD

**Issue by ayman-2468** [jrchamp/moodle-mod_zoom#568]
I installed the zoom plugin in moodle when trying to establish a connection I get the following message

Unable to find your account on Zoom. If you are using Zoom for the first time, you must activate your Zoom account by logging into https://zoom.us/profile. Once you've activated your Zoom account, reload this page and continue setting up your meeting. Else make sure your email on Zoom matches your email on this system.

**Comment by jrchamp** [comment]
Is the Moodle user's email address set to the same email address as the Zoom user? The Zoom user will need to be part of the overall Zoom account that was used to create the Zoom Marketplace Server-to-Server OAuth App.

**Comment by ayman-2468** [comment]
Yes the Moodle user's email address is the same address as the Zoom account

**Comment by jrchamp** [comment]
Is the Moodle user's Zoom account part of the Zoom Server-to-Server OAuth app's account?

**Comment by michael-milette** [comment]
I can easily reproduce this issue when the current user is not listed in the Zoom account.

However, my small issue with this is the Continue button below the message. When you click it, it I expected that it would take me back to the course's main page. But instead, it just reloads the page.

Suggestion: Change the name of the button from "Continue" to "Try again". Also, add a second button called "Cancel" which takes you back to the course page.

Best regards,

Michael
